packages/react/src/input/Input.tsx

```diff
@@ -9,16 +9,25 @@ import * as styles from "./Input.css";
 type InputProps = ExtendProps<
   ComponentPropsWithRef<"input">,
   ComponentPropsWithRef<typeof Box>,
-  { isDisabled?: boolean; isInvalid?: boolean } & styles.InputVariants
+  {
+    isDisabled?: boolean;
+    isInvalid?: boolean;
+    leftSection?: ReactNode;
+    rightSection?: ReactNode;
+  } & styles.WrapperVariants
 >;
 
 export const Input = forwardRef<HTMLInputElement, InputProps>(
   (
     {
       className,
+      defaultValue,
+      id,
       isDisabled,
```

Comment:
let's rename `isDisabled` to just `disabled` - since that's the actual attribute name. that way we're following the general convention, not making it a surprise to users, and making it consistent with other components like `Button`.

Reply:
Sure vai